tox.ini

```diff
@@ -0,0 +1,59 @@
+[tox]
+envlist = py37
+isolated_build = true
+skipsdist=True
+
+toxworkdir={toxinidir}/.tox
+
+[testenv]
+deps=
+    -r {toxinidir}/requirements/common.txt
+    -r {toxinidir}/requirements/dev.txt 
+commands_pre = 
+    pip install -r {toxinidir}/requirements/docs.txt
+    apt-get install shellcheck
+commands =
+    pip check
+    {toxinidir}/lints/queuelint.py
+    flake8 --show-source
+    isort --check-only --diff --quiet
+    mkdocs build
+    sh -c "git grep -El '^#!/.+\b(bash|sh)\b' | xargs shellcheck"
```

Comment:
Run this instead: `pre-commit run --all-files shellcheck`

This removes the need to install `shellcheck`.